### PR TITLE
Fix file command in IBM X-Force Exchange (v2)

### PR DESCRIPTION
**Pull request by adi88d**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23154

## Description
The command !file crashed because the response structure of `/malware/{filehash}` was different than expected, for example 'Family' value should gets from `report['origins']['external']['family]` instead of `report['family']`.

<ins>raw response example:</ins>
```
{
   "origins":{
      "external":{
         "source":"reversingLabs",
         "firstSeen":"2018-02-15T07:32:42Z",
         "lastSeen":"2019-03-22T19:50:59Z",
         "malwareType":"Trojan",
         "platform":"Win32",
         "detectionCoverage":87,
         "family":[
            "emotet"
         ]
      }
   },
   "type":"sha256",
   "sha256":"0x9C0E1E",
   "hash":"0x9C0E1E",
   "risk":"high"
}
```

Link to documentation:
https://exchange.xforce.ibmcloud.com/api/doc/?#Malware_get_malware_filehash

## Screenshots
before:
image

after:
image

### CODE REVIEW

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ configuration:
   type: 0
 description: IBM X-Force Exchange lets you receive threat intelligence about applications,
   IP addresses, URls and hashes
-display: IBM X-Force Exchange (v2)
+display: IBM X-Force Exchange v2
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.yml]: as long as you fix the integration, please update line 58 in this file (remove trailing space and fix typo):
should be `description: score threshold`

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py [hunk 2]
```diff
@@ -379,13 +379,18 @@ def file_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, Any]:
 
         file_key = f'XFE.{outputPaths["file"]}'
 
-        hash_info = {**report['origins'], 'Family': report['family'], 'FamilyMembers': report['familyMembers']}
+        report_data = report['origins'].get('external', {})
+        family_value = report_data.get('family')
+
+        hash_info = {**report['origins'], 'Family': ', '.join(family_value),
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py]: why did you switch this?
```suggestion
        hash_info = {**report['origins'], 'Family': family_value,

```

File: Packs/XForceExchange/Integrations/XFE_v2/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Fixed an issue in the ***!file*** command.
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/CHANGELOG.md]:
```suggestion
Fixed an issue in the ***file*** command. 
```

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py (hunk 2 above)
Comment (adi88d) [reply Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py]: family value is array, you can see example in the PR description

Comment (yaakovi) [reply Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py]: Leave it as list

**Comment by kirbles19** [comment]
@adi88d Done.

**Pull request merged**